app/models/stolen_record.rb

```diff
@@ -241,4 +245,43 @@ def find_or_create_recovery_link_token
     update_attributes(recovery_link_token: SecurityTokenizer.new_token)
     recovery_link_token
   end
+
+  def location
```

Comment:
There are already a number of address functions in stolen record, which are necessary for legacy reasons.

Is there a way to make this either use one of them, or consolidate them in some other way? Or unify it with the location method in bike? AKA I think you could do a little refactoring to make some of the address grossness nicer than anything I've done 🙂

Reply:
Renamed in 1bb09e6d to follow suit with the existing methods, but I'm not seeing a way to build on those without munging their output, which seems like needless coupling

Reply:
Fair. All I could ask 😉